bot/exts/recruitment/talentpool/_review.py

```diff
@@ -36,11 +36,14 @@
 MAX_EMBED_SIZE = 4000
 
 # Regex for finding the first message of a nomination, and extracting the nominee.
-# Historic nominations will have 2 role mentions at the start, new ones won't, optionally match for this.
 NOMINATION_MESSAGE_REGEX = re.compile(
-    r"(?:<@&\d+> <@&\d+>\n)*?<@!?(\d+?)> \(.+#\d{4}\) for Helper!\n\n\*\*Nominated by:\*\*",
+    r"<@!?\d+> \((.+)#\d{4}\) for Helper!\n\n\*\*Nominated by:\*\*",
     re.MULTILINE
 )
+if DEBUG_MODE:
+    NOMINATION_THREAD_ARCHIVE_TIME = ThreadArchiveTimes.DAY.value
+else:
+    NOMINATION_THREAD_ARCHIVE_TIME = ThreadArchiveTimes.WEEK.value
```

Comment:
This setting should maybe just be part of the config, or at least be computed in `bot/constants.py` to keep configuration logic out of here. What do you think?

Reply:
Hmm, I guess we could have a `DEFAULT_THREAD_ARCHIVE_TIME` in constants.py yea.

Reply:
https://github.com/python-discord/bot/pull/1928/commits/b8d393e90e550e425327314ddf72c6275718f688